scripts/output_as_hack.js

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env node --harmony
+
+var program = require('commander');
+
+program
+  .version('0.7.0')
```

Comment:
I'm not sure what best practice is, but is it worth importing `package.json` and just doing `package.version` here?